packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -244,3 +246,15 @@ export default defineComponent({
   },
 })
 </script>
+
+<style lang="scss" scoped>
```

Comment:
```suggestion
<style lang="scss">
```
we don't want to use `scoped` styles in components

Reply:
Changed to global classes.